apps/web/lib/video/[uid]/getServerSideProps.ts

```diff
@@ -161,6 +191,11 @@ export async function getServerSideProps(context: GetServerSidePropsContext) {
       calVideoLogo,
       displayLogInOverlay,
       loggedInUserName: sessionUserId ? session?.user?.name : undefined,
+      showRecordingButton,
+      redirectUrlOnExit:
+        sessionUserId !== bookingObj.user?.id
```

Comment:
nit: We should give the condition a name like `isNotBookingOrganizer` or similar.

Reply:
Also shouldn't we consider other hosts of the booking in case of collective booking e.g.